README.md

```diff
@@ -24,7 +24,7 @@
 
 # 📖 [Awesome Books] <a name="about-project"></a>
 
-**[Awesome Books]** is a basic website that allows users to add/remove books to/from a list of books.
+**[Awesome Books]** is a basic website that allows users to add/remove books to/from a list of books. This is a simple website that displays a list of books and allows you to add and remove books from that list. By building this application, I will learn how to manage data using JavaScript. That's why the website will be more interactive. I will also use a medium-fidelity wireframe to build the UI.
```

Comment:
- [ ] Please update the LICENSE section of your [README](https://github.com/CollinsTatang/readme-template) file the current link is broken.
These changes will help make your README file look professional before the eyes of recruiters.